src/badge/demos/enUS/index.demo-entry.md

```diff
@@ -19,17 +19,17 @@ raw
 
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
-| color | `string` | `undefined` |  |
-| dot | `boolean` | `false` |  |
-| max | `number` | `undefined` |  |
-| processing | `boolean` | `false` |  |
-| show-zero | `boolean` | `false` |  |
-| show | `boolean` | `true` |  |
-| type | `'default' \| 'success' \| 'error' \| 'warning' \| 'info'` | `'default'` |  |
-| value | `string \| number` | `undefined` |  |
+| color | `string` | `undefined` | Badge color. |
+| dot | `boolean` | `false` | Show badge as dot. |
+| max | `number` | `undefined` | TThe maximum number of the badge when its value overflows. |
+| processing | `boolean` | `false` | Show processing status. |
+| show-zero | `boolean` | `false` | Badge show zero. |
```

Comment:
Another cryptic message. I would change it to: _Whether to display the badge, even if provided value equals 0_